src/Hexagons.jl

```diff
@@ -375,21 +375,7 @@ function cube_round(x, y, xsize=1.0, ysize=1.0)
     q = sqrt(3)/3 * x - y/3
     r = 2 * y / 3
     h = nearest_cubic_hexagon(q, -q - r, r)
-    #return h
-
-    x0, y0 = center(h)
-    d0 = (x0-x)^2 + (y0-y)^2
-    h_best = h
-    d_best = d0
-    for neighbor in neighbors(h)
-        xn, yn = center(neighbor)
-        dn = (xn-x)^2 + (yn-y)^2
-        if dn < d_best
-            d_best = dn
-            h_best = neighbor
-        end
-    end
-    h_best
```

Comment:
As far as I can tell, this section is no longer necessary with `center`'s default offset at zero, not 1. The redblobgames references doesn't describe this neighbor check code. Seems like this latter loop was previously necessary because all hexagons were shifted by `(-1,-1)` in Cartesian space, so the shifted hexagon that contains a Cartesian point might not be the unshifted hexagon that contains that point.

With this shift behavior removed, `nearest_cubic_hexagon` is already returning the correct (0-centered) hexagon for the input point.